src/main/java/core/model/misc/Verein.java

```diff
@@ -76,6 +99,10 @@ public Verein(Properties properties) throws Exception {
         m_iFinancialDirectorLevels = DF.parse(properties.getProperty("financialdirectorlevels", "0")).intValue();
         m_iFormCoachLevels = DF.parse(properties.getProperty("formcoachlevels", "0")).intValue();
         m_iTacticalAssistantLevels = DF.parse(properties.getProperty("tacticalassistantlevels", "0")).intValue();
+        m_iGlobalRanking = DF.parse(properties.getProperty("globalranking", "0")).intValue();
+        m_iLeagueRanking = DF.parse(properties.getProperty("leagueranking", "0")).intValue();
+        m_iRegionRanking = DF.parse(properties.getProperty("regionranking", "0")).intValue();
+        m_iPowerRating = DF.parse(properties.getProperty("powerrating", "0")).intValue();
```

Comment:
If those properties come from the HRF file, there could be an issue here, as the keys seem camel case when converting XML to HRF:
https://github.com/akasolace/HO/pull/529/files#diff-f57a7f84ee6b7283a69deee3ea8d9764R362-R365

Reply:
actually the get property calls toLowerCase (or something equivalent).  I tested it and it seems to work (values end up in database)